source/common/util/is-attachment.js

```diff
@@ -17,14 +17,11 @@ const isFile = require('./is-file')
 /**
  * This function determines whether or not a given path describes an attachment.
  * @param  {string}  p The path to be checked.
+ * @param  {Boolean} skipExistenceCheck Whether or not to skip the existence check
  * @return {Boolean}   Returns true, if the path is an attachment, or false.
  */
-module.exports = function (p) {
+module.exports = function (p, skipExistenceCheck = false) {
   let ext = global.config.get('attachmentExtensions')
-  if (!ext) {
-    // Something went wrong on init. Hey ZettlrConfig, are you even there?
-    return false
-  }
-
-  return isFile(p) && ext.includes(path.extname(p).toLowerCase())
+  let fileExists = skipExistenceCheck ? true : isFile(p)
```

Comment:
ext and fileExists could be const

Reply:
Hey, I saw your comment over on reddit, thanks for reviewing this! I've been playing with the thought to `const` such variables myself, but I want to do this on the whole repository in one piece. But you are right!